### PR TITLE
feat: fixed window playback

### PR DESCRIPTION
**Pull request by pauldambra**
When viewing a recording, if there are multiple screens we guesstimate which is the active window and show you that one.

In the past when we've got that wrong folk have asked for the ability to choose one window to watch

Today when viewing a user's bug report I'd have loved to know if we were picking the active window wrong or something else... So, let's let people pick

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx [hunk 1]
```diff
@@ -120,6 +130,29 @@ export function PlayerMeta(): JSX.Element {
         )
     }
 
+    const windowOptions: LemonSelectOption<string | null>[] = [
+        {
+            label: <IconWindow value={currentWindowIndex} className="text-muted-alt" />,
+            value: null,
+            labelInMenu: (
+                <div className="flex flex-row gap-2 space-between items-center">
+                    Follow the user: <IconWindow value={currentWindowIndex} className="text-muted-alt" />
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:139]: Wasn't sure of the language on these labels...

Could this just be "Auto"??

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx [hunk 2]
```diff
@@ -120,6 +130,25 @@ export function PlayerMeta(): JSX.Element {
         )
     }
 
+    const windowOptions: LemonSelectOption<string | null>[] = [
+        {
+            label: <IconWindow value={currentWindowIndex} className="text-muted-alt" />,
+            value: null,
+            labelInMenu: <>Follow the user</>,
```
Comment (daibhin) [inline frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:137]: It looked during the gif like this option also had an `IconWindow` in it. That somewhat confused me but I'm guessing it has changed

**Review by daibhin — APPROVED** [review]

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx (hunk 1 above)
Comment (daibhin) [reply frontend/src/scenes/session-recordings/player/PlayerMeta.tsx]: I like follow the user!

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:137]: yes, i realised it didn't make sense :)

**Comment by pauldambra** [comment]
closes #18150

**Pull request merged**